bundles/python/python_lexer.moon

```diff
@@ -46,12 +46,16 @@ howl.aux.lpeg_lexer ->
   operator = c 'operator', S'+-*/%~&^=!<>;:,.(){}[]|`'
 
   name = (alpha + '_')^1 * (alpha + digit + P'_')^0
+  type_name = c 'type', (upper^1 * (alpha + digit + '_')^0)
```

Comment:
@shalabhc Actually, I got an idea.

Python [PEP 484](https://www.python.org/dev/peps/pep-0484/) adds type annotations.

The function annotations syntax is always like:

```
('(' | ',') identifier [':' annotation]
```

So, maybe a rule could be added like:

```moonscript
S'(,' * name * ':' * type_name
```

where `type_name` would be either one of the built-int types (e.g. `str`, `int`, `bytes`) *or* something matching the above rule that you put.